src/index.js

```diff
@@ -1,6 +1,16 @@
 const p = console.log;
 const puts = console.log;
 
+Number.prototype.times = function(callback) {
+  for (let i = 0; i < +this; i++) {
+    callback(i);
+  }
+  return;
+};
+
+// usage :
+// new Number(2).times(console.log);
```

Comment:
I think the usage of this function should be something like this
```
(2).times( () => console.log("Hello") );
```
Also, I'm thinking about adding a file with examples of usage, so please delete the commented lines

Reply:
Thanks, I'll change that